python/webdriver_test.py

```diff
@@ -27,20 +27,25 @@
 git push origin master
 
 """
+import pytest
 from selenium import webdriver
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 
-def test_01():
-    b = webdriver.Firefox()
+@pytest.fixture
+def browser(request):
+    driver = webdriver.Firefox()
+    def teardown():
+        driver.quit()
+    request.addfinalizer(teardown)
+    return driver
+
+def test_01(browser):
+    b = browser
```

Comment:
The idea is to keep code a bit shorter, but i see your point. Switched webdriver naming to "browser", it certainly improves code readability. Thanks.